### PR TITLE
Improve: option 'cases-exp-indent' to adjust indent

### PR DESCRIPTION
**Pull request by gpetiot**
From JS fork (https://github.com/janestreet/ocamlformat/commit/9225436db1aa09009a4cce0f4581753b99f1b74f)

### CODE REVIEW

**Comment by hhugo** [comment]
Is there an ocp-indent config option that we could reuse?

**Comment by hhugo** [comment]
I don't like the `ocpi_` prefix, we might as well be more explicit `-ocp-indent-match-clause`.
Also it might be a good time to look at https://github.com/ocaml-ppx/ocamlformat/issues/290

**Comment by gpetiot** [comment]
Agreed, I will start looking into this one real soon.

**Comment by gpetiot** [comment]
I think using a prefix allows to differentiate the ocamlformat name (and best name in this case) from the ocp-indent name. Having two different option names without a prefix would be confusing for the user I guess. Do you think it is better to only use the ocp-indent option name?

**Comment by jberdine** [comment]
I'm not sure I understand the intended use cases, so take this with a 
grain of salt. But I think I would have added unprefixed ocamlformat 
options that made sense on their own, and then reading an ocp-indent 
config file would set the corresponding ocamlformat options.

**Comment by gpetiot** [comment]
So options should only be set on the command line using their ocamlformat name, but reading ocp-indent options from ocp-indent configuration files should set the corresponding ocamlformat option ?

**Comment by gpetiot** [comment]
I removed the ocp-indent alternative option name, ocp-indent options are handled by #445

File: ocamlformat-help.txt [hunk 1]
```diff
@@ -56,6 +56,10 @@ OPTIONS (CODE FORMATTING STYLE)
            unconditionally. natural will break struct-end phrases naturally
            at the margin. The default value is force.
 
+       --cases-exp-indent=COLS
+           Indentation of cases expressions (COLS columns). The default value
```
Comment (hhugo) [inline ocamlformat-help.txt]: There is an exception to this. It is always 2 for nested match/try

**Pull request merged**